tempesta_fw/t/functional/leaks/test_leak.py

```diff
@@ -0,0 +1,110 @@
+"""
+Testing for memory leaks
+"""
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2017 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+import os.path
+import subprocess
+
+import psutil
+from time import sleep
+
+import unittest
+from testers import stress
+from helpers import tf_cfg, control, tempesta
+
+
+class LeakTest(stress.StressTest):
+        """ Leaks testing """
```

Comment:
Please, run `pylint2` tool before sending patches. the tool reports of multiple warnings and errors on this file.